packages/wrangler/src/__tests__/jest.setup.ts

```diff
@@ -55,3 +55,17 @@ jest.mock("../dialogs");
     "Unexpected call to `prompt()`. You should use `mockPrompt()` to mock calls to `prompt()` with expectations. Search the codebase for `mockPrompt` to learn more."
   );
 });
+
+jest.mock("../dev", () => {
+  //eslint-disable-next-line @typescript-eslint/no-var-requires
```

Comment:
FWIW you can avoid these eslint rules by using `jest.requireActual()` rather than `require()`.